src/fixtures/DemoData.tsx

```diff
@@ -0,0 +1,4 @@
+import maureenDemo from 'fixtures/MaureenMcodeDemoPatientRecords.json';
+import maureenDemo2 from 'fixtures/MaureenMcodeDemoPatientRecords2.json';
+
+export const DemoRecords = [maureenDemo, maureenDemo2];
```

Comment:
Again this definitely works, but there's the downside of we have to make code changes every time we want to add a new demo file. Maybe it's not possible with the files being in the `src` folder, but could you look into doing a `fetch` for patient records under the `public/static` folder, similar to what we do for CQL libraries? We could enforce a filename pattern or just say file `maureenDemo.json` gets read when you go to `http://pathways/#/demo/maureenDemo`

Reply:
refactored to fetch records from public/static